lib/el3_runtime/aarch32/cpu_data.S

```diff
@@ -34,7 +34,9 @@ endfunc _cpu_data
  * -----------------------------------------------------------------
  */
 func _cpu_data_by_index
+	mov_imm	r1, #CPU_DATA_SIZE
```

Comment:
The build failure is because `mov_imm` doesn't need the `#` for the immediate data. Same comment for the AArch64 file as well.

Reply:
oh, sorry...